src/hooks/table.js

```diff
@@ -0,0 +1,182 @@
+import { useState } from 'react';
+import debounce from 'lodash/debounce';
+import filter from 'lodash/filter';
+import pickBy from 'lodash/pickBy';
+import keys from 'lodash/keys';
+import keyBy from 'lodash/keyBy';
+import mapValues from 'lodash/mapValues';
+import identity from 'lodash/identity';
+import intersection from 'lodash/intersection';
+
+import { SEARCH_DEBOUNCE_DELAY } from '../constants';
+
+const chain = (f1, f2) => (...args) => {
+    f1(...args);
+    f2(...args);
+};
+
+function callbacks () {
```

Comment:
Phew his function is really hard o understand can you add some comments what does it do?

Reply:
@karelhala thanks for feedback. I added a comment